### ISSUE TITLE
Counter event change example does no longer work without type

### ISSUE THREAD

**Issue by danielpeintner** [eclipse/thingweb.node-wot#697]
While experimenting with another PR  I run into some strange behavior.

The counter.html sample can listen to change events

![grafik](https://user-images.githubusercontent.com/11281461/158147907-e9999332-c9aa-4262-9050-5ab4b29bb01c.png)

If I remove `data: { type: "integer" }` it does no longer work. VERY strange!

https://github.com/eclipse/thingweb.node-wot/commit/f68c03144d068333637063f460a006d1285c488e#r68629000

**Comment by relu91** [comment]
> If I remove data: { type: "integer" } it does no longer work. VERY strange!

Yeah, I experienced the same problem that's why I added the type there. The problem is that the data function NEEDs a data type to work properly...

**Comment by danielpeintner** [comment]
The problem with the sample is fixed with https://github.com/eclipse/thingweb.node-wot/pull/698
